File: elementary_computer_science/C#/arithmetic_digit_number.cs
using System;
namespace Creative1 {
	class arithmetic_digit_number {
		static void Main(string[] args) {
			Show(Find());
			Console.WriteLine("\n fini");
			Console.ReadLine();
		}

		static int[] Find() {
			int cnt = 0;
			int [] odd_digit = {1, 3, 5, 7, 9};
			int [] s = new int[25];
			int x;
			for (int i = 0; i < 5; ++i) {
				x = 105 * odd_digit[i];
				for (int j = 0; j < 5; ++j)
					s[++cnt] = x + 6 * odd_digit[j];
			}
			return s;
		}

		static void Show(int[] s) {
			foreach (int x in s) Console.Write(x + " ");
		}
	}
}

File: elementary_computer_science/Pascal/arithmetic_digit_number.pas
program arithmetic_digit_number;
uses crt;
const odd_digit: array [1..5] of integer = (1, 3, 5, 7, 9);
var s: array [1..25] of integer;
	n: integer;

function find: integer;
var a, c, cnt, x: integer;
begin
	cnt := 0;
	for a := 1 to 5 do
	begin
		x := 105 * odd_digit[a];
		for c := 1 to 5 do
		begin
			inc(cnt);
			s[cnt] := x + 6 * odd_digit[c];
		end;
	end;
	find := cnt;
end;

procedure display(n: integer);
var i: integer;
begin
	for i := 1 to n do write(s[i]:4);
end;

begin
	n := find;
	display(n);
	writeln;
	writeln('Total ', n, ' numbers.');
end.